charts/redisoperator/Chart.yaml

```
@@ -1,4 +1,15 @@
+annotations:
+  category: Redis Operator
+appVersion: 3.1.0
```

redis-operator is currently version 1.0.0
```suggestion
appVersion: 1.0.0
```